README.md

```diff
@@ -19,67 +19,69 @@ Additional expect matchers for http clients (e.g. Axios), supports `jest`, `vite
-    - [.toHaveTemporaryRedirectStatus()](#tohavetemporaryredirectstatus)
-    - [.toHavePermanentRedirectStatus()](#tohavepermanentredirectstatus)
-    - [.toHaveBadRequestStatus()](#tohavebadrequeststatus)
-    - [.toHaveUnauthorizedStatus()](#tohaveunauthorizedstatus)
-    - [.toHavePaymentRequiredStatus()](#tohavepaymentrequiredstatus)
-    - [.toHaveForbiddenStatus()](#tohaveforbiddenstatus)
-    - [.toHaveNotFoundStatus()](#tohavenotfoundstatus)
-    - [.toHaveMethodNotAllowedStatus()](#tohavemethodnotallowedstatus)
-    - [.toHaveNotAcceptableStatus()](#tohavenotacceptablestatus)
-    - [.toHaveProxyAuthenticationRequiredStatus()](#tohaveproxyauthenticationrequiredstatus)
-    - [.toHaveRequestTimeoutStatus()](#tohaverequesttimeoutstatus)
-    - [.toHaveConflictStatus()](#tohaveconflictstatus)
-    - [.toHaveGoneStatus()](#tohavegonestatus)
-    - [.toHaveLengthRequiredStatus()](#tohavelengthrequiredstatus)
-    - [.toHavePreconditionFailedStatus()](#tohavepreconditionfailedstatus)
-    - [.toHaveRequestTooLongStatus()](#tohaverequesttoolongstatus)
-    - [.toHaveRequestUriTooLongStatus()](#tohaverequesturitoolongstatus)
-    - [.toHaveUnsupportedMediaTypeStatus()](#tohaveunsupportedmediatypestatus)
-    - [.toHaveRequestedRangeNotSatisfiableStatus()](#tohaverequestedrangenotsatisfiablestatus)
-    - [.toHaveExpectationFailedStatus()](#tohaveexpectationfailedstatus)
-    - [.toHaveImATeapotStatus()](#tohaveimateapotstatus)
-    - [.toHaveInsufficientSpaceOnResourceStatus()](#tohaveinsufficientspaceonresourcestatus)
-    - [.toHaveMethodFailureStatus()](#tohavemethodfailurestatus)
-    - [.toHaveMisdirectedRequestStatus()](#tohavemisdirectedrequeststatus)
-    - [.toHaveUnprocessableEntityStatus()](#tohaveunprocessableentitystatus)
-    - [.toHaveLockedStatus()](#tohavelockedstatus)
-    - [.toHaveFailedDependencyStatus()](#tohavefaileddependencystatus)
-    - [.toHaveUpgradeRequiredStatus()](#tohaveupgraderequiredstatus)
-    - [.toHavePreconditionRequiredStatus()](#tohavepreconditionrequiredstatus)
-    - [.toHaveTooManyRequestsStatus()](#tohavetoomanyrequestsstatus)
-    - [.toHaveRequestHeaderFieldsTooLargeStatus()](#tohaverequestheaderfieldstoolargestatus)
-    - [.toHaveUnavailableForLegalReasonsStatus()](#tohaveunavailableforlegalreasonsstatus)
-    - [.toHaveInternalServerErrorStatus()](#tohaveinternalservererrorstatus)
-    - [.toHaveNotImplementedStatus()](#tohavenotimplementedstatus)
-    - [.toHaveBadGatewayStatus()](#tohavebadgatewaystatus)
-    - [.toHaveServiceUnavailableStatus()](#tohaveserviceunavailablestatus)
-    - [.toHaveGatewayTimeoutStatus()](#tohavegatewaytimeoutstatus)
-    - [.toHaveHttpVersionNotSupportedStatus()](#tohavehttpversionnotsupportedstatus)
-    - [.toHaveInsufficientStorageStatus()](#tohaveinsufficientstoragestatus)
-    - [.toHaveNetworkAuthenticationRequiredStatus()](#tohavenetworkauthenticationrequiredstatus)
+    - [Generic Matchers](#generic-matchers)
```

Comment:
I would split to status first

Reply:
What do you mean by that, do you mean you want only the status to be split for now and the rest being a part of the API section?

Reply:
yes